api_tests/src/actors/actor.ts

```diff
@@ -489,7 +488,7 @@ export class Actor {
                 defaultAssetDescription(asset, ledger)
             );
 
-            expect(currentWalletBalance).to.be.bignumber.gte(balanceInclFees);
+            expect(currentWalletBalance.gte(balanceInclFees)).toBeTruthy();
```

Comment:
:mega: This is going to produce horribly unhelpful error messages (false is not equal to true).